### ISSUE TITLE
How to use Font Awesome Icons properly

### ISSUE THREAD

**Issue by WilsonCazarre** [electron-react-boilerplate/electron-react-boilerplate#2432]
I'm building a simple SPA with this boilerplate. I got a sidebar for navigate trough my app and I'm using Font Awesome Icons React library for it icons. The icons works fine in dev mode, put when run in production, they just doesn't show up.

This is the file I'm using to load the icons:
app/helpers/fontawesome.js

```
import { library } from '@fortawesome/fontawesome-svg-core';
import { fas } from '@fortawesome/free-solid-svg-icons';
library.add(fas);
```

and this is a example of component that I'm using:

```
import { IconName } from '@fortawesome/fontawesome-svg-core';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import React from 'react';
import '../../helpers/fontawesome';
import styles from './CircleIcon.css';

type Props = {
  iconName: IconName;
  isSelected?: boolean;
};

export default function CircleIcon(props: Props) {
  const { iconName, isSelected } = props;
  let style = styles.circleButton;

  if (isSelected) {
    style += ` ${styles.selected}`;
  }
  return (
    <div className={styles.container}>
      <div className={style}>
        <FontAwesomeIcon icon={['fas', iconName]} size="2x" />
      </div>
    </div>
  );
}
```

I already tried load the library from both package.json. Same issue.
So basically I want to understand why this works only on dev mode.

**Comment by ndom91** [comment]
I'm not sure if this is related, but I had trouble using assets like pictures, etc. in the production build as well. I had to manually add the paths my assets were in to the `files` array under `build` in the package json. That tells `electron-builder` to include the assets in the final build. Maybe font-awesome needs some assets manually included as well, that aren't by default in the prod build. (like css, svgs, etc.)

**Comment by niksauer** [comment]
Here's my setup:

**global.ts**

**index.tsx**
```
import './global';
import App from './App';

const AppContainer = process.env.PLAIN_HMR ? Fragment : ReactHotAppContainer;

document.addEventListener('DOMContentLoaded', () =>
  render(
    <AppContainer>
      <App />
    </AppContainer>,
    document.getElementById('root')
  )
);
```
